src/components/checkbox/Checkbox.tsx

```diff
@@ -35,6 +36,17 @@ export class Checkbox extends React.Component<ICheckboxProps, any> {
         }
     }
 
+    private handleOnClick(e: React.MouseEvent<HTMLElement>) {
+        if (!!!this.props.disabled) {

```

Comment:
3 `!` does this have a special meaning? If you only meant not disabled, only one `!` will do.

Reply:
I can receive an undefined disabled, with this 3!, I'm sure I will have a condition boolean

Reply:
It does not matter, it's an if `!this.props.disabled` and `!!!this.props.disabled` will always have the exact same result here.